examples/sprites/tests/bindings/test_sprites.swift

```diff
@@ -1,5 +1,8 @@
 import sprites
 
+let sempty = Sprite(initialPosition: nil)
+assert( sempty.getPosition() == Point(x: 0, y: 0))
```

Comment:
This is making me think more about argument labeling in Swift—labeling all arguments feels pretty wordy, especially when the type and purpose are obvious, but `Sprite(initialPosition: nil)` reads much better than `Sprite(nil)`.